src/bootctl/bootctl-status.c

```diff
@@ -160,7 +160,7 @@ static int print_efi_option(uint16_t id, int *n_printed, bool in_order) {
         printf("    Partition: /dev/disk/by-partuuid/" SD_ID128_UUID_FORMAT_STR "\n",
                SD_ID128_FORMAT_VAL(partition));
         printf("         File: %s%s%s/%s%s\n",
-               glyph(GLYPH_TREE_RIGHT), ansi_grey(), arg_esp_path, ansi_normal(), path);
+               glyph(GLYPH_TREE_RIGHT), ansi_grey(), strna(arg_esp_path), ansi_normal(), path);
```

Comment:
It doesn't make sense to print a path with "n/a" in it. In what scenario do you see the NULL here?